src/control_flow/mod.rs

```diff
@@ -252,6 +287,8 @@ impl Visit for Analyzer<'_> {
 
     if let Some(done) = self.scope.done {
       self.mark_as_done(s.span.lo, done);
+    } else {
+      self.mark_as_done(s.span.lo, Done::Pass);
```

Comment:
Maybe you can rename

 - `Done` => `End`
 - `Pass` => `Continue`
 - `mark_as_done` => `mark_as_end`

I'm not sure which one is better.
cc @bartlomieju What do you think about this?

Reply:
Suggested names seem more descriptive and self-explanatory to me 👍

Reply:
> Done => End
> Pass => Continue
> mark_as_done => mark_as_end

Looks nice to me, too. I'll rename them.